generate_eval_train.py

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+import pathlib
+import sys
+
+
+def split_file(input_file, ratio):
+    """
+    Splits a text file into list.train and  list.eval with lines ratio.
+    """
+    if not isinstance(input_file, pathlib.Path):
+        input_file = pathlib.Path(input_file)
+    if not input_file.exists():
+        print(f"'{input_file}' not exists!")
+        return False
+    lines = input_file.read_text().splitlines()
+
+    split_point = int(ratio * len(lines))
+    output_dir = input_file.resolve().parent
+    train_list = pathlib.Path(output_dir, 'list.train')
+    eval_list = pathlib.Path(output_dir, 'list.eval')
+
+    with open(train_list, 'w', newline='\n') as f1, open(
+        eval_list, 'w', newline='\n'
+    ) as f2:
+        f1.writelines(lines[:split_point])
+        f2.writelines(lines[split_point:])
```

Comment:
```suggestion
    with open(train_list, 'w', newline='\n') as f:
        f.writelines(lines[:split_point])
    with open(eval_list, 'w', newline='\n') as f:
        f.writelines(lines[split_point:])
```

Reply:
I think it looks simpler like that. Is there a reason to implement it differently?

Reply:
As far as I see modern examples and tutorials on the internet: if you open more files with `with` they are always combined into one as in my PR.
Also, python linters do not like single-letter (`f`) variables and for readability, I suggest using different names for eval and train files.